docs/patterns/Auth/Integrations.mdx

```diff
@@ -83,12 +145,17 @@ print(connection_request.redirectUrl)
 ```
 </CodeGroup>
 
-### Whitelabeling the OAuth Developer App Using Integrations 
+### Using Your Own OAuth Developer App
 
-Integrations can be used to customise the OAuth Developer App that is shown to the user when initiating a connection.
+When creating an integration, you can use your own OAuth Developer App instead of Composio's default one, you can follow the same steps as mentioned in [Creating a New Integration](#creating-a-new-integration) and:
```

Comment:
In the Integrations.mdx file, consider maintaining consistency in spelling: "customise" should be "customize" to match American English usage throughout the documentation.